### PR TITLE
[prometheus-pushgateway] bump to 1.8.0

### PR DESCRIPTION
**Pull request by DrFaust92**
#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-pushgateway/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "v1.7.0"
+appVersion: "v1.8.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 2.8.0
+version: 2.9.0
```
Comment (zeritti) [inline charts/prometheus-pushgateway/Chart.yaml:5]:
```suggestion
version: 2.10.0
```

**Review by zeritti — CHANGES_REQUESTED** [review]
Thank you, @DrFaust92, for your PR. Please, raise the chart version.

File: charts/prometheus-pushgateway/Chart.yaml (hunk 1 above)
Comment (DrFaust92) [reply charts/prometheus-pushgateway/Chart.yaml:5]: @zeritti but its bumped, from 2.8.0 to 2.9.0

Comment (DrFaust92) [reply charts/prometheus-pushgateway/Chart.yaml:5]: I see now, nvm

**Review by zeritti — APPROVED** [review]

**Pull request merged**